### PR TITLE
DLP For Mac Updates

### PR DESCRIPTION
**Pull request by danwdavey**
Line 403: either needs to be ammended to state Mac OS DLP for Endpoint, or removing. Confirmed "Both Outlook mobile/ and Outlook Mac do not support policy tips today."

Line: 396: Outlook Mac should be its own line item

### CODE REVIEW

File: microsoft-365/compliance/dlp-policy-tips-reference.md [hunk 1]
```diff
@@ -393,7 +393,8 @@ Please note that custom sensitive information types will also be detected in add
...
-|**Outlook Mobile (iOS, Android)/Outlook Mac**|:::image type="icon" source="../media/crsmrk.png" border="false":::|none|none|DLP policy tips are not supported on Outlook mobile|
+|**Outlook Mobile (iOS, Android)**|:::image type="icon" source="../media/crsmrk.png" border="false":::|none|none|DLP policy tips are not supported on Outlook mobile|
+|**Outlook Mac**|:::image type="icon" source="../media/crsmrk.png" border="false":::|none|none|DLP policy tips are not supported on Outlook mobile|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-tips-reference.md]:
```suggestion
|**Outlook for Mac**|:::image type="icon" source="../media/crsmrk.png" border="false":::|none|none|DLP policy tips are not supported on Outlook for Mac|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @danwdavey, kindly commit the minor suggested change. Also, please change line 404 to "Endpoint DLP policies are...".Many thanks.

**Comment by chrfox** [comment]
@danwdavey  -Thanks for your comment.  I am confirming with PG before publishing

File: microsoft-365/compliance/dlp-policy-tips-reference.md (hunk 1 above)
Comment (danwdavey) [reply microsoft-365/compliance/dlp-policy-tips-reference.md]: I think it should state 'DLP policy tips are not supported on Outlook for Mac' ?

**Comment by danwdavey** [comment]
> @danwdavey -Thanks for your comment. I am confirming with PG before publishing

@chrfox - Added comment above, edit needs to state Outlook for Mac, not outlook for Mobile

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by danwdavey** [comment]
The article is still wrong;
DLP policy tips are not supported on Outlook mobile

This **should** read: 

DLP policy tips are not supported on Outlook Mac



<br class="Apple-interchange-newline">
</body>
</html>

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@danwdavey, could you please commit the suggested change to make the correction?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
final query out to Swati Kumari to see if this is correct

**Pull request merged**